packages/core/helper-plugin/lib/src/components/DateTimePicker/tests/index.test.js

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import { render, screen } from '@testing-library/react';
+import { ThemeProvider } from '@strapi/design-system/ThemeProvider';
+import { lightTheme } from '@strapi/design-system/themes';
+import DateTimePicker from '../index';
+
+describe('DateTimePicker', () => {
+  it('snapshots the component', () => {
+    const { container } = render(
+      <ThemeProvider theme={lightTheme}>
+        <DateTimePicker
+          value={new Date('2021-10-13T10:00:00.000Z')}
+          onChange={() => {}}
+          name="datetimepicker"
+          label="Date time picker"
+          hint="This is a super description"
+        />
+      </ThemeProvider>
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  it('should initialize the two inputs', () => {
```

Comment:
Can you add a bit more tests? Like one to make sure that it keeps the initial seconds.

Reply:
We don't keep the initial seconds. The timepicker will select the nearest value in the dropdown. We already talked about this fix during the TimePicker fix @JabStrapi

Reply:
That's why we keep the step 1 for the moment